<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-P7GG8UH" failures="1" tests="3" name="API_Assignment" time="8.176" errors="0" timestamp="2023-01-08T10:20:08 IST">
  <testcase classname="API_test" name="POST" time="5.093"/>
  <testcase classname="API_test" name="GET" time="1.696"/>
  <testcase classname="API_test" name="PUT" time="1.271">
    <failure type="java.lang.AssertionError" message="expected [200] but found [403]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [403]
at API_test.PUT(API_test.java:114)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- PUT -->
</testsuite> <!-- API_Assignment -->
